test/TestApps/PushNotificationsTestApp/main.cpp

```diff
@@ -153,16 +160,18 @@ bool RegisterActivatorNullDetails()
 // Verify calling register activator with null clsid is not allowed.
 bool RegisterActivatorNullClsid()
 {
-    PushNotificationManager::UnregisterActivator(PushNotificationRegistrationActivators::PushTrigger | PushNotificationRegistrationActivators::ComActivator);
-    g_registeredActivator = false;
-
+    PushNotificationManager::UnregisterAllActivators();
     try
     {
-        PushNotificationActivationInfo info(
-            PushNotificationRegistrationActivators::PushTrigger | PushNotificationRegistrationActivators::ComActivator,
-            winrt::guid()); // Null guid
+        // Only applies to packaged apps
+        if (g_isPackaged)
```

Comment:
A test shouldn't decide whether it wants to run or not.

This isn't the only test that is only valid for packaged or for unpackaged apps, and other such tests don;t try to decide whether they should run or not.

Reply:
Actually, we should probably have a test that verifies that this test fails the way we expect them for an unpackaged app

Reply:
It's interesting because the clsid is unused in the unpackaged scenario. So it would be failure for one case and success for another. Should we just have two separate tests for pkg/unpkg -> failure/success?

Reply:
that's how I would do it.

Reply:
Switching to IsActivatorSupported and leaving current impl.